huntsman/utils/config.py

```diff
@@ -144,3 +146,47 @@ def query_config_server(key=None, name='config_server'):
             
 #==============================================================================
 
+def load_device_config(key=None, config_files=None, logger=None, **kwargs):
+    '''
+    Load the device config from either the config server or local files.
+    
+    Parameters
+    ----------
+    key:
+        The key used to query the config server. Only used if config_files
+        is None.
+    config_files:
+        List of config file names. If None (default), use the config server.
+        
+    Returns
+    -------
+    dict:
+        The config dictionary.
+    '''
+    if logger is None:
+        logger = DummyLogger()
+        
+    #Load config from local files?
+    if config_files is not None:
+        logger.debug(f'Loading config from local file(s).')
+        try:
+            config = load_config(config_files, **kwargs)
+        except Exception as e:
```

Comment:
As this really just calls https://github.com/panoptes/POCS/blob/develop/pocs/utils/config.py#L11 which has extensive error handling, I don't think you need a try/except here as its redundant